Python/ast_opt.c

```diff
@@ -7,12 +7,23 @@
 #include "pycore_setobject.h"     // _PySet_NextEntry()
 
 
+/* See PEP 765 */
 typedef struct {
+    bool in_finally;
+    bool in_funcdef;
+    bool in_loop;
+} ControlFlowInFinallyState;
```

Comment:
```suggestion
} ControlFlowInFinallyContext;
```
This represents the context in which we are operating